restli-client/src/test/java/com/linkedin/restli/internal/client/TestQueryParamsUtil.java

```diff
@@ -133,4 +133,21 @@ public void testCustomProjectionDataMapSerializerReturningNull()
     Assert.assertNull(dataMap.getDataMap(RestConstants.PAGING_FIELDS_PARAM));
     Assert.assertNull(dataMap.getDataMap(RestConstants.METADATA_FIELDS_PARAM));
   }
+
+  @Test
+  public void testPreSerializedProjectionParams()
+  {
+    Map<String, Object> queryParams = new HashMap<>();
+    queryParams.put(RestConstants.FIELDS_PARAM, "fields");
```

Comment:
I wonder if we could add more cases into testing, 
for example (excerpt from restli doc)
https://linkedin.github.io/rest.li/Projections
```
  map_field:($*:(field1),key1:(field2),key2:(field3))
  array_field:($*:(field1,field2),$start:10,$count:15)
```
and projections like
```
  http://host:port/context/resource/id?fields=person:(firstname,lastname)
```

Reply:
We should already have tests for MaskTree decoder covering these. Need not test them again here.

Reply:
+1 on no testing here.

Reply:
Yep already tested, resolving.